src/frontend/screens/Library/components/CategoriesManager/index.tsx

```diff
@@ -30,6 +30,7 @@ function CategoryItem({
   const [renameMode, setRenameMode] = useState(false)
   const [newName, setNewName] = useState(name)
   const [removeMode, setRemoveMode] = useState(false)
+  const isNewNameEmptyOrEqualsOldName = newName === '' || newName === name
```

Comment:
just curious, does this prevent blank strings like a string full of white spaces too?

Reply:
Good point. No, it doesn't prevent it.
I will change this.

Reply:
Done.